src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Microsoft.WinFx.props

```diff
@@ -1,16 +0,0 @@
-<Project>
-  <PropertyGroup>
```

Comment:
Moved to WinFx.target

Reply:
Instead of moving the code, just add import into WinFX targets, if they are not already imported.

OR Is that pattern not preferred?

Reply:
In this case, props will be imported regardless if the workload is installed or not. So we need to move everything that could be moved.

For more see the description, linked issue. As well as the doc related to the net5.0 design.

Reply:
You could keep the logic in the props file and import it in the Sdk targets. No need to move the logic and risk losing history. -- Just a suggestion